thirdparty/runtime_environments.go

```diff
@@ -365,13 +365,14 @@ func stringToTime(timeInitial string) (time.Time, error) {
 	return time.Unix(timestamp, 0), nil
 }
 
-// Image stores information about an image including its name, version ID, kernel, AMI, and its last deployed time.
+// Image stores information about an image including its AMI, imageID, kernel, last deployed time, name, and versionID.
```

Comment:
nit: 'image ID' and 'version ID' in comments 
comments should generally just be human readable things